tests/Unit/CrudPanel/CrudPanelButtonsTest.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+namespace Backpack\CRUD\Tests\Unit\CrudPanel;
+
+use Backpack\CRUD\PanelTraits\CrudButton;
+
+class CrudPanelButtonsTest extends BaseCrudPanelTest
+{
+    private $defaultButtonNames = [
+        'preview', 'update', 'revisions', 'delete', 'create', 'reorder',
+    ];
+
+    private $topViewButton;
+    private $lineViewButton;
+    private $bottomViewButton;
+    private $topModelFunctionButton;
+
+    protected function setUp()
```

Comment:
vs `setup` i.e. no capital U?

Reply:
> PHPUnit supports sharing the setup code. Before a test method is run, a template method called setUp() is invoked. setUp() is where you create the objects against which you will test. Once the test method has finished running, whether it succeeded or failed, another template method called tearDown() is invoked. tearDown() is where you clean up the objects against which you tested.

From the [PHPUnit manual](https://phpunit.de/manual/current/en/fixtures.html).

Reply:
Yes, but is it really EVILLY spelled with a **capital U**?

Reply:
OK, sure, you can override methods using a different case style, because PHP.
I'm not saying we would be the only ones doing it, but we would certainly be in the minority.

What is our interest in breaking the standard here?

Reply:
Oh, upstream spells it that way. *mumble grumble*